heat/core/statistics.py

```diff
@@ -1151,10 +1278,51 @@ def std(x, axis=None, ddof=0, **kwargs):
         return exponential.sqrt(var(x, axis, ddof, **kwargs), out=None)
 
 
+def __torch_skew(torch_tensor, dim=None, unbiased=False):
+    # calculate the sample skewness of a torch tensor
+    # return the bias corrected Fischer-Pearson standardized moment coefficient by default
+    if dim is not None:
+        n = torch_tensor.shape[dim]
+        diff = torch_tensor - torch.mean(torch_tensor, dim=dim, keepdim=True)
+        m3 = torch.true_divide(torch.sum(torch.pow(diff, 3), dim=dim), n)
+        m2 = torch.true_divide(torch.sum(torch.pow(diff, 2), dim=dim), n)
+    else:
+        n = torch_tensor.numel()
+        diff = torch_tensor - torch.mean(torch_tensor)
+        m3 = torch.true_divide(torch.sum(torch.pow(diff, 3)), n)
+        m2 = torch.true_divide(torch.sum(torch.pow(diff, 2)), n)
+    if not unbiased:
+        return torch.true_divide(m3, torch.pow(m2, 1.5))
+    coeff = ((n * (n - 1)) ** 0.5) / (n - 2.0)
+    return coeff * torch.true_divide(m3, torch.pow(m2, 1.5))
+
+
+def __torch_kurtosis(torch_tensor, dim=None, Fischer=True, unbiased=False):
```

Comment:
Can we have parameter and return type annotations here (except DNDarray due to aliasing)